src/types/collection.graphql

```diff
@@ -5,6 +5,8 @@ type Collection {
   accessConstraints: JSON
   "The data’s distinctive attributes of the collection (i.e. attributes used to describe the unique characteristics of the collection which extend beyond those defined)."
   additionalAttributes: JSON
+  "The list of concepts and any data on the relationship on between this collection and other permitted concepts"
```

Comment:
```suggestion
  "The list of concepts and any data on the relationship between this collection and other permitted concepts"
```

Reply:
I'll fix this one in the next ticket since I need to change this file anyway for order-options